tests/components/elmax/test_config_flow.py

```diff
@@ -221,9 +221,21 @@ async def test_no_online_panel(hass):
 
 async def test_show_reauth(hass):
     """Test that the reauth form shows."""
+    entry = MockConfigEntry(
+        domain=DOMAIN,
+        data={
+            CONF_ELMAX_PANEL_ID: MOCK_PANEL_ID,
+            CONF_ELMAX_USERNAME: MOCK_USERNAME,
+            CONF_ELMAX_PASSWORD: MOCK_PASSWORD,
+            CONF_ELMAX_PANEL_PIN: MOCK_PANEL_PIN,
+        },
+        unique_id=MOCK_PANEL_ID,
+    )
+    entry.add_to_hass(hass)
+
     result = await hass.config_entries.flow.async_init(
         DOMAIN,
-        context={"source": SOURCE_REAUTH},
+        context={"source": SOURCE_REAUTH, "entry_id": entry.entry_id},
```

Comment:
```suggestion
        context={"source": SOURCE_REAUTH, "unique_id": entry.unique_id, "entry_id": entry.entry_id},
```
Also below

Reply:
Done in 59b5c7f6e6cbebd005e9b74b73da31fa5612cb94, thanks!